### DISCUSSION TITLE
DevOps Engineer "Portfolio"

### DISCUSSION THREAD

**Discussion by T1taN101** [community/community#65736] [Programming Help]
### Select Topic Area

Question

### Body

Hey everyone,
What projects can I make and push into Github for my portfolio as a DevOps Engineer?

Thanks ahead.

**Comment by of1r** [comment]
You can try [this](https://github.com/Abhinav-26/DevOps-Projects) repository for project ideas

**Comment by Binary-Ninja-007** [comment]
Good.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
